vendor/licenses/git_submodule/K-VSCode.dep.yml

```
@@ -0,0 +1,31 @@
+---
+name: K-VSCode
+version: c6d017b7225d3877e60edb4af5463cc624cd6d7d
+type: git_submodule
+homepage: https://github.com/LucianCumpata/K-VSCode.git
+license: other
```

Can override this because the auto-detection hasn't been able to determine it.

```suggestion
license: mit
```